tests/model_servering/server/llm/conftest.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+from kubernetes.dynamic import DynamicClient
+from kubernetes.dynamic.exceptions import NotFoundError
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.namespace import Namespace
+from ocp_resources.resource import ResourceEditor, get_client
+from ocp_resources.secret import Secret
+from ocp_resources.service_account import ServiceAccount
+from ocp_resources.service_mesh_member_roll import ServiceMeshMemberRoll
+from ocp_resources.serving_runtime import ServingRuntime
+from pytest_testconfig import config as py_config
+
+from tests.model_servering.server.llm.utils import base64_encode_str
+from utilities.constants import TRANSFORMERS_CACHE_ENV_VAR
+from utilities.inference_service import InferenceServiceForTests
+
+
+@pytest.fixture(scope="session")
+def admin_client() -> DynamicClient:
+    return get_client()
+
+
+@pytest.fixture(scope="class")
+def model_namespace(request, admin_client: DynamicClient) -> Namespace:
+    with Namespace(
+        client=admin_client,
+        name=request.param["name"],
```

Comment:
Would it make sense to have a default name, in case it is not passed in the request?

Reply:
having a default ns will lead to a problem identifying which test created it, in case of left overs